src/actions/togglerActions.js

```diff
@@ -0,0 +1,4 @@
+import { createAction } from 'redux-actions'
+import { TOGGLER } from './constants'
+
```

Comment:
sorry, in my example I called counter "counterActions.js," but it should really be just called "counter.js". 

actions/counterActions or actinos/togglerActions is a bit redundant.